packages/fluentui/docs/src/prototypes/Calls/recentContact.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { Avatar, Flex, Text, Menu } from '@fluentui/react-northstar';
+import { CallIcon, CallVideoIcon } from '@fluentui/react-icons-northstar';
+
+const Contact: React.FC<{
+  image?: string;
+  name: string;
+  status: object;
+}> = ({ image, name, status }) => {
+  return (
+    <Flex fill gap="gap.small" space="between">
+      <div>
+        <Avatar image={`data:image/jpeg;base64,${image}`} name={name} status={status} />
+        <Text content={name} variables={{ isNameText: true }} />
+      </div>
+      <Menu
+        iconOnly
+        items={[
+          {
+            icon: <CallIcon outline />,
```

Comment:
Should we add some actions on these elements?

Reply:
What actions do you have in mind?